_layouts/home.html

```diff
@@ -1,70 +1,295 @@
-              <div class="front front-background"
-                style="background-image: url(https://images.unsplash.com/photo-1569683795645-b62e50fbf103?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987&q=80); background-size: cover;">
-                <div class="card-body py-7 text-center">
-                  <i class="material-icons text-white text-4xl my-3">touch_app</i>
-                  <h3 class="text-white">Download the Competency Framework <br /> Document Kit</h3>
-                  <p class="text-white opacity-8">All the skills and learning pathsways are available in the kit
-                    take a look.</p>
+        <div class="col-lg-6">
+          <div class="row py-5">
+            <h3>Welcome</h3>
+            <p class="text-lg">Welcome to The RSE Competencies Toolkit, a community resource to support RSEs (Research
+              Software Engineers) in tracking and managing their professional development. It is currently in the early
+              stages of development and we welcome contributions and feedback.</p>
+            <p>Note that, while we've focussed on RSE during the early stages of development, this tool can be used for
+              any roles. We plan to extend this to other research roles and welcome contributions and collaboration from
+              anyone working in research, broadly defined.</p>
+            <p>Please join our <a href="https://ukrse.slack.com/archives/C05CY0YFWEL">Slack Channel</a> to connect and
+              discuss your ideas or suggestions with other members of the group.</p>
+            <p><a href="{% link about.md %}" class="text-info icon-move-right">More about the project
+                <i class="fas fa-arrow-right text-sm ms-1" aria-hidden="true"></i>
+              </a>
+            </p>
+          </div>
+        </div>
+        <div class="col-lg-4 ms-auto mt-lg-0 mt-4">
+          <div class="card">
+            <div class="card-header p-0 position-relative mt-n4 mx-3 z-index-2">
+              <a class="d-block blur-shadow-image">
+                <img
+                  src="https://images.unsplash.com/photo-1544717302-de2939b7ef71?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80"
+                  alt="img-colored-shadow" class="img-fluid border-radius-lg">
+              </a>
+            </div>
+            <div class="card-body text-center">
+              <h5 class="font-weight-normal">
+                <a href="javascript:;">A framework for success</a>
+              </h5>
+              <p class="mb-0">
+                The Competencies Framework is a community model that defines the knowledge, skills, and behaviours
+                that individuals need to attain
+                to succeed in research software roles
+              </p>
+              <button type="button" class="btn bg-gradient-info btn-sm mb-0 mt-3">Draft Framework</button>
```

Comment:
This could link to https://github.com/RSEToolkit/rse-competencies-toolkit/blob/main/_data/skills.json - in the long run I expect we'll change it but better to have a working link right now

Reply:
Done